dynparquet/schema.go

```diff
@@ -611,6 +611,35 @@ func (s Schema) parquetSchema(
 	}
 }
 
+// parquetSortingSchema returns the parquet schema of just the sorting columns
+// with the concrete dynamic column names given in the argument.
+func (s Schema) parquetSortingSchema(
```

Comment:
cc @thorfour since this schema is just used for sorting I assume there's nothing to do extra for handling nested schemas?